Makefile

```diff
@@ -43,5 +43,11 @@ in_vagrant:
 on_host:
 	@if [ ${IN_VAGRANT} -eq 1 ]; then echo "*** Run on host ***"; exit 1; fi
 
+locale:
+	mkdir -p locale/$(LOCALE)/LC_MESSAGES && \
```

Comment:
Can you add a `@` in front of the mkdir call to make make not print the command before running it?